src/main/resources/transformation/metafix-wip.fix

```diff
@@ -0,0 +1,27 @@
+/*
+  <entity name="TitleOther">
+    <entity name="title[]" flushWith="record">
+      <data name="" source="246?[ 345678].a">
+        <replace pattern="\.$" with="" />
+      </data>
+    </entity>
+  </entity>
+	*/
+
+# TODO: Das schreiben in einen tieferliegendes Array funktioniert anscheinend nicht.
+
+# set_array("TitleOther.title[]")
+# do list(path: "246?[ 345678]", "var": "varTitleOther")
+#	 replace_all("varTitleOther.a", "\\.$", "")
+#	 copy_field("varTitleOther.a", "TitleOther.title[].$append")
+# end
+
+set_array("TO[]")
+do list(path: "246?[ 345678]", "var": "varTitleOther")
+	 replace_all("varTitleOther.a", "\\.$", "")
```

Comment:
Could you use two spaces for indentation?

Reply:
Don't know why this slipped in, two spaces is on default. Will have an eye on this.